R/add_activity_after_death_flag.R

```diff
@@ -0,0 +1,184 @@
+#' Match on BOXI NRS death dates to process activity after death flag
+#'
+#' @description Match on CHI number where available in the episode file, and add date of death from the BOXI NRS lookup.
+#' Create new activity after death flag
+#'
+#' @param data episode files
+#' @param year financial year, e.g. '1920'
+#' @param deaths_data The death data for the year
+#'
+#' @return data flagged if activity after death
+add_activity_after_death_flag <- function(
+    data,
+    year,
+    deaths_data = read_file(get_all_slf_deaths_lookup_path())) {
+  death_joined <- data %>%
+    dplyr::select(year, chi, record_keydate1, record_keydate2, death_date, deceased) %>%
```

Comment:
```suggestion
    dplyr::select(.data$year, .data$chi, .data$record_keydate1, .data$record_keydate2, .data$death_date, .data$deceased) %>%
```
I think we need `.data$` throughout where we are using variables

Reply:
or if this has depreciated with one of the versions of tidyverse then we need quotes in place